### PR TITLE
Release v2.5.1

### PR DESCRIPTION
**Pull request by UncleClapton**
* Move billing info from shipping info stage to order summary page
* Handle cases where the user changes their cart mid-order
* Introduce `sortPriority` metadata field for store products, letting us determine the view order.
* Clean up currency string formation by defining a helper function to do it for us. (Get Money ( ͡° ͜ʖ ͡°))
* Hide attribute descriptors if no attributes for a product exist
* Make it more clear that our billing form is secured through stripe
* Clean up old cruft we don't use
* Remove packages which are no longer used
* Remove all i18n stuff as we have since decided not to translate the website
* Rename instances of `store` subdirectories to `storefront`. This is to avoid confusion between the redux store and the stripe storefront
* Clean up server modules, and only disable certain eslint rules instead of simply turning off eslint entirely for certain modules.
* Clean up getActionOptions even further, and rename to buildActionOptions 
* Forward dispatch and getState functions to action response handlers.
* define an actionStatus enum type.
  * this is currently only being used in the actionCreators file, however the plan is to convert all string comparisons to enum comparisons so eventually we can turn the strings into symbols.
* rewrite blog retrieval logic AGAIN so we can fully remove isomorphic-fetch from the project

### CODE REVIEW

File: helpers/getMoney.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// ( ͡° ͜ʖ ͡°) because xlexious told me to
```
Comment (SuperManifolds) [inline helpers/getMoney.js:1]: ( ͡° ͜ʖ ͡°)

File: server/config/router.js [hunk 2]
```diff
@@ -76,47 +39,52 @@ module.exports = function (nextjs, koa) {
...
       day,
       month,
       slug,
       year,
     } = ctx.params
 
-    let dayIsValid = parseInt(day) && (day.length === 2)
-    let monthIsValid = parseInt(month) && (month.length === 2)
-    let yearIsValid = parseInt(year) && (year.length === 4)
+    const dayIsValid = parseInt(day, 10) && (day.length === 2)
```
Comment (SuperManifolds) [inline server/config/router.js:49]: Why are you using safeParseInt in some places but not here?

**Review by SuperManifolds — COMMENTED** [review]
Only issue I've found is with the parseInt stuff

File: server/config/router.js (hunk 2 above)
Comment (UncleClapton) [reply server/config/router.js:49]: cuz thats a server end thing to help resolve old permalinks. I'm not too worried about it as I don't generally expect user-generated input to pass through it. that and this function scares me

Comment (SuperManifolds) [reply server/config/router.js:49]: sounds good to me

File: helpers/getMoney.js (hunk 1 above)
Comment (UncleClapton) [reply helpers/getMoney.js:1]: ( ͡° ͜ʖ ͡°)

**Review by SuperManifolds — APPROVED** [review]
LGTM

**Pull request merged**